services/base.spec.js

```diff
@@ -320,7 +368,7 @@ describe('BaseService', function() {
   })
 
   describe('ScoutCamp integration', function() {
-    const expectedRouteRegex = /^\/foo\/([^/]+).(svg|png|gif|jpg|json)$/
+    const expectedRouteRegex = /^\/foo\/([^/]+?)\.(svg|png|gif|jpg|json)$/
```

Comment:
1. Today I learned that `+?` is a [lazy qualifier](https://javascript.info/regexp-greedy-and-lazy), which causes the segment to match as little as possible. It doesn't make this any less correct, and I imagine makes it easier to build pattern regexes more reliably.
2. `.` to `\.` fixes a minor bug. Before this change, we accept any character as the extension separator.